src/main/Config.cpp

```diff
@@ -1766,6 +1799,19 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
             throw std::runtime_error(msg);
         }
 
+#ifdef BUILD_TESTS
+        if (TESTING_MAX_CLASSIC_BYTE_ALLOWANCE +
```

Comment:
I think we should only run this check for non-zero allowances, since otherwise they just get ignored. Or maybe we can just use the getters directly for this check.